xtask/src/commands/build.rs

```diff
@@ -0,0 +1,62 @@
+use crate::constants::*;
+use clap::{arg, ArgMatches, Command};
+use console::style;
+use duct::cmd;
+use std::fs;
+
+pub(crate) fn build_command() -> Command {
```

Comment:
I think it is easier to use the derive functionality of clap for defining and parsing your command.

Reply:
A non-flag option is just adding it without `short` or `long`. In terms of reducing bouilerplate code, now you are just moving that in the command handler itself, instead of just depending on a strongly typed struct.